exporter/metric/metric.go

```diff
@@ -505,6 +506,16 @@ func (me *metricExporter) recordToTspb(m metricdata.Metrics, mr *monitoredrespb.
 			ts.Metric = me.recordToMpb(m, point.Attributes, library, extraLabels)
 			tss = append(tss, ts)
 		}
+	case metricdata.ExponentialHistogram[float64]:
```

Comment:
Based on https://github.com/open-telemetry/opentelemetry-go/blob/a7034da63185be247206ed9539d8ab93d982d7e3/exporters/otlp/otlpmetric/otlpmetricgrpc/internal/transform/metricdata.go#L218, it looks like the only difference between int64 and float64 exponential histograms is the sum.  Can we accept int64 exponential histograms as well, and just cast the sum to a float64?

Reply:
Actually, it looks like you already handle int64 exponential histograms below.  I think you just need to handle the `case metricdata.ExponentialHistogram[int64]:` here.

Reply:
Added `case` for `int64` type. Fixed.